package.json

```diff
@@ -142,6 +142,7 @@
     "@storybook/addon-storysource": "^5.3.3",
     "@storybook/addons": "^5.3.3",
     "@storybook/react": "^5.3.3",
+    "@visual-snapshot/jest": "^1.0.8",
```

Comment:
Where is the code for this? I couldn't find much about it on npm.

Reply:
Ah let me put up a repo -- deciding if all this stuff should be in a new gh org or in a mono-repo

Reply:
https://github.com/getsentry/jest-visual-snapshot